### ISSUE TITLE
Brienne is listed as dead

### ISSUE THREAD

**Issue by kajo404** [Rostlab/JS16_ProjectF#421]
but she has no date of death

**Comment by yashha** [comment]
Its because we don't get any plods for her:
https://api.got.show/api/plod/bySlug/Brienne_of_Tarth

**Comment by goldbergtatyana** [comment]
Yeah, strangely, she is not in the list of characters with a predicted PLOD. Group 6, @ThuyNganTran @subburamr @asesselmann, any idea why there is no prediction for her? Thanks!

**Comment by sacdallago** [comment]
@goldbergtatyana isn't it better if they show nothing in case of no PLOD for that char? For dead chars, we have 100% PLOD, right? So no PLOD is different from dead

**Comment by goldbergtatyana** [comment]
This would make a lot of sense, yes! very good idea @sacdallago 

In the last group email Guy wrote that we processed 2028 characters, while Group 6 provided predictions for 1946 characters.

This makes a difference of 82 characters for whom we don't have a PLOD, so yeah it would be great if we could catch them and say "sorry no likelihood of death available" or so.

@mammuth @yashha @jorjo1 could we have this feature?

**Comment by subburamr** [comment]
Yes, I noticed that Brienne of Tarth is not in the 1946 characters. So no plod data is present. We will check and update why there is a difference.

**Comment by goldbergtatyana** [comment]
Thanks @subburamr for the super quick reply! We realized that we are missing PLODs for 82 characters. Could you please then look into those as well, why they escaped being trained with our ML algorithm? :)

**Comment by subburamr** [comment]
We seem to have missed these characters somewhere in the data processing, will update once we find the cause.

Here is the list of 82 characters which were not included in the training.
[82_chars_missing_plods.txt](https://github.com/Rostlab/JS16_ProjectF/files/217705/82_chars_missing_plods.txt)

Comparison with the 2028 characters from api.
[character_differences.txt](https://github.com/Rostlab/JS16_ProjectF/files/217706/character_differences.txt)

**Comment by goldbergtatyana** [comment]
Yeay, great we have the list @subburamr !

Do you also have the 26 features for them? If so, please run your machine learning algorithm in the exact same way with the exact same parameters on the full data set now (1946 + 82 characters) and report the performance here please. 

:+1: :+1:

**Comment by subburamr** [comment]
It appears that project A added the 82 characters after Mar 25th(our deadline) :smile: 
https://github.com/Rostlab/JS16_ProjectA/issues/105

I will rerun the algorithm with the extra characters and upload the performance.

**Comment by gyachdav** [comment]
make sure that the prediction you already made do not change :laughing:

**Comment by mammuth** [comment]
What's the status here?
@subburamr

**Comment by subburamr** [comment]
I added the results [here](https://github.com/Rostlab/JS16_ProjectB_Group6/issues/66#issuecomment-210808627)

**Comment by mammuth** [comment]
Cool! 

I'm not aware of the workflow how this is going to be inserted into the A-Database, could someone bring light into this or make this going happen? @sacdallago 

As far as I understood, this is going to resolve this issue, right?

**Comment by sacdallago** [comment]
I am, yes

**Comment by marcusnovotny** [comment]
@sacdallago This keeps coming up on reddit and twitter. Fix needed :pray:
